source/guides/clear/mixer.rst

```diff
@@ -287,9 +313,58 @@ set to get a smaller kernel image, which will also be faster to load.
    Look in ~/mixer/update/www/<mix version> to see the update content in your
    workspace.
 
-#. Configure image. Edit the ister configuration file for your image to include
-   all of the bundles you want preinstalled in the image. If this is the first
-   time creating an image, first get a copy of the
+Example 3: Create an update for your mix
+========================================
+
+Next let’s create a new version of the mix. We’ll add a new package.
+
+#. Make the next mix. Create a new version of your mix, for the live image to
+   update to. Increment your mix version by 10:
+
+   .. code-block:: bash
+
+      mixer versions update
+
+#. Add the upstream :command:`curl` bundle to version 20 of the mix:
+
+   .. code-block:: bash
+
+      mixer bundle add curl
+
+#. Build your next mix version that incorporates the new bundle.
+
+   .. code-block:: bash
+
+      mixer build bundles
+      mixer build update
+
+Build delta-packs
+-----------------
+
+Optionally, you can build delta-packs, which helps reduce client update time:
+
+.. code-block:: bash
+
+   mixer build delta-packs --from 10 --to 20
+
+Refresh your \http://localhost site to see the update content for
+mix version 20.
+
+Look in ~/mixer/update/www/<mix version> to see the update content in your
+workspace.
+
+
+Example 4: Build an image
+=========================
+
+This example shows how to build an image incorporating the "kernel-kvm”
+bundle from  Example 2. ister is an independent tool that can be used for building an image.
```

Comment:
IMO, "ister is an independent tool that can be used for building an image." should be removed unless we go into details on how to use ister manually.  Otherwise, the reader is just left hanging.  :)

Reply:
The main reason why I wanted this here was to avoid getting ister bug reports on mixer's github. Would it work if we had a link to ister's github page in here? 
https://github.com/bryteise/ister

Reply:
Done.